zh/restore-from-pv-using-br.md

```diff
@@ -0,0 +1,149 @@
+---
+title: 使用 BR 工具恢复持久卷上的备份数据
+summary: 介绍如何使用 BR 工具将存储在持久卷上的备份数据恢复到 TiDB 集群
+---
+
+# 使用 BR 工具恢复持久卷上的备份数据
+
+本文描述了如何将存储在[持久卷](https://kubernetes.io/zh/docs/concepts/storage/persistent-volumes/)上的备份数据恢复到 Kubernetes 环境中的 TiDB 集群。底层通过使用 [`BR`](https://docs.pingcap.com/zh/tidb/dev/backup-and-restore-tool) 来进行集群恢复。
+
+本文描述的持久卷指任何 [Kubernetes 支持的持久卷类型](https://kubernetes.io/zh/docs/concepts/storage/persistent-volumes/#types-of-persistent-volumes)。以下示例以 NFS 存储卷为例，介绍如何将存储在持久卷上指定路径的集群备份数据恢复到 TiDB 集群。
+
+本文使用的恢复方式基于 TiDB Operator 新版（v1.1.8 及以上）的 CustomResourceDefinition (CRD) 实现。
+
+## 数据库账户权限
+
+- `mysql.tidb` 表的 `SELECT` 和 `UPDATE` 权限：恢复前后，restore CR 需要一个拥有该权限的数据库账户，用于调整 GC 时间
+
+> **注意：**
+>
+> 如果使用 TiDB Operator >= v1.1.7 && TiDB >= v4.0.8, BR 会自动调整 `tikv_gc_life_time` 参数，该步骤可以省略。
+
+## 环境准备
+
+1. 下载文件 [`backup-rbac.yaml`](https://github.com/pingcap/tidb-operator/blob/master/manifests/backup/backup-rbac.yaml)，并执行以下命令在 `test2` 这个 namespace 中创建恢复所需的 RBAC 相关资源：
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl apply -f backup-rbac.yaml -n test2
+    ```
+
+2. 创建 `restore-demo2-tidb-secret` secret，该 secret 存放用来访问 TiDB 服务的账号的密码：
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl create secret generic restore-demo2-tidb-secret --from-literal=user=root --from-literal=password=<password> --namespace=test2
+    ```
+
+    > **注意：**
+    >
+    > 如果使用 TiDB Operator >= v1.1.7 && TiDB >= v4.0.8, BR 会自动调整 `tikv_gc_life_time` 参数，该步骤可以省略。
+
+3. 确认可以从 Kubernetes 集群中访问用于存储备份数据的 NFS 服务器。
+
+## 恢复过程
+
+1. 创建 restore custom resource (CR)，将指定的备份数据恢复至 TiDB 集群：
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl apply -f restore.yaml
+    ```
+
+    `restore.yaml` 文件内容如下：
+
+    ```yaml
+    ---
+    apiVersion: pingcap.com/v1alpha1
+    kind: Restore
+    metadata:
+      name: demo2-restore-nfs
+      namespace: test2
+    spec:
+      # backupType: full
+      br:
+        cluster: demo2
+        clusterNamespace: test2
+        # logLevel: info
+        # statusAddr: ${status-addr}
+        # concurrency: 4
+        # rateLimit: 0
+        # checksum: true
+      # # Only needed for TiDB Operator < v1.1.7 or TiDB < v4.0.8
+      # to:
+      #   host: ${tidb_host}
+      #   port: ${tidb_port}
+      #   user: ${tidb_user}
+      #   secretName: restore-demo2-tidb-secret
+      local:
+        prefix: backup-nfs
```

Comment:
This is not explained in the following descriptions.

Reply:
this doc refers to backup doc in L101